integration-cli/docker_cli_run_unix_test.go

```diff
@@ -1584,3 +1585,38 @@ func (s *DockerSuite) TestRunWithNanoCPUs(c *check.C) {
 	assert.ErrorContains(c, err, "")
 	c.Assert(out, checker.Contains, "Conflicting options: Nano CPUs and CPU Period cannot both be set")
 }
+
+func (s *DockerSuite) TestRunWithContainerdLabels(c *check.C) {
```

Comment:
Can you move this test to the `integration/` test-suite? `integration-cli` has been deprecated;

```
17:41:56 The following new tests were added to integration-cli:
17:41:56 
17:41:56 +func (s *DockerSuite) TestRunWithContainerdLabels(c *check.C) {
17:41:56 
17:41:56 integration-cli is deprecated. Please add an API integration test to
17:41:56 ./integration/COMPONENT/. See ./TESTING.md for more details.
17:41:56 
```

Reply:
Ya the tricky part is itn's not exposed via the API but I'll see what I can do.